test/versioned/aws-sdk-v3/package.json

```diff
@@ -209,6 +210,20 @@
         "bedrock-embeddings.test.js",
         "bedrock-negative-tests.test.js"
       ]
+    },
+    {
+      "engines": {
+        "node": ">=18.0"
+      },
+      "dependencies": {
+        "@aws-sdk/client-redshift-data": {
+          "versions": ">=3.0.0",
```

Comment:
this may be going back too far. when i run `npm run versioned:internal aws-sdk-v3` if fails on 3.0.0. Maybe just test on `>=3.474.0` like we do in bedrock client